src/types/Submissions.d.ts

```diff
@@ -18,7 +18,27 @@ type DataSubmission = {
   updatedAt: string; // # ISO 8601 date time format with UTC or offset e.g., 2023-05-01T09:23:30Z
 };
 
-type DataSubmissionStatus = "New" | "In Progress" | "Submitted" | "Released" | "Withdrawn" | "Rejected" | "Completed" | "Archived" | "Canceled";
+type SubmissionHistoryEvent = {
```

Comment:
This is the same thing as the line below: `type DataSubmissionHistoryEvent = HistoryBase<SubmissionStatus>;`
See HistoryBase type for more details.